ui-react/src/components/topBar/LogsCanvas.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import Offcanvas from "react-bootstrap/Offcanvas";
+
+type LogsCanvasProps = {
+  show: boolean;
+  handleClose?: () => {};
+};
+
+export default function LogsCanvas({ show, handleClose }: LogsCanvasProps) {
+  return (
+    <Offcanvas
+      show={show}
+      onHide={handleClose}
+      placement="bottom"
+      style={{ zIndex: 9999 }}
```

Comment:
Oh, good!

Simple but essential!

But, Is there no built-in dialog component in the library?

Reply:
I'm using the _react-bootstrap_ built-in Modal, but also a OffCanvas for the logs list.
And the latter should be on top of the former, which forces me to deal with z-indices.

Reply:
so I created zIndexesManager.ts which manages the z-indexes layers.
Not sure if I did it right + had to add " //@ts-ignore" since I got some typescript problem there.